src/Engine/Serialization/LegacyImages.h

```diff
@@ -638,42 +638,42 @@ void Deserialize(const MonsterDesc_MM7 &src, MonsterDesc *dst);
 
 struct Actor_MM7 {
     std::array<char, 32> pActorName;
-    int16_t sNPC_ID;
-    int16_t field_22;
-    uint32_t uAttributes;
-    int16_t sCurrentHP;
+    int16_t sNPC_ID{};
+    int16_t field_22{};
+    uint32_t uAttributes{};
+    int16_t sCurrentHP{};
```

Comment:
Btw I believe I've left these w/o initialization for a reason. We're memsetting them anyway.